dirac-distribution/Dockerfile

```diff
@@ -30,12 +30,14 @@ RUN echo "   GSSAPIDelegateCredentials yes" >> /etc/ssh/ssh_config && \
     echo "   StrictHostKeyChecking no" >> /etc/ssh/ssh_config
 
 # Sencha + ExtJS
-ADD http://cdn.sencha.com/cmd/7.0.0.40/no-jre/SenchaCmd-7.0.0.40-linux-amd64.sh.zip .
-RUN unzip SenchaCmd-7.0.0.40-linux-amd64.sh.zip
-RUN chmod +x SenchaCmd-7.0.0.40-linux-amd64.sh
-RUN ./SenchaCmd-7.0.0.40-linux-amd64.sh -q
-ADD http://cdn.sencha.com/ext/gpl/ext-6.2.0-gpl.zip .
-RUN unzip ext-6.2.0-gpl.zip
+RUN curl -LO http://cdn.sencha.com/cmd/7.0.0.40/no-jre/SenchaCmd-7.0.0.40-linux-amd64.sh.zip . && \
+    unzip SenchaCmd-7.0.0.40-linux-amd64.sh.zip && \
+    chmod +x SenchaCmd-7.0.0.40-linux-amd64.sh && \
+    ./SenchaCmd-7.0.0.40-linux-amd64.sh -q && \
+    curl -LO http://cdn.sencha.com/ext/gpl/ext-6.2.0-gpl.zip . && \
+    unzip ext-6.2.0-gpl.zip && \
+    rm SenchaCmd-7.0.0.40-linux-amd64.sh.zip ext-6.2.0-gpl.zip
```

Comment:
what was wrong with using the `ADD` command instead of `curl`?

Reply:
It adds layers in the image making it slower to pull. This way the zip files can be cleaned up before the layer is committed.

I can revert if you'd prefer?

Reply:
you also had to add curl for this to run. If you don't mind, I prefer the previous way...

Reply:
Adding curl is nothing in comparison to the savings from removing the source archives (compressed image is 332MB smaller, 912MB → 580MB) but I'll revert.